backend/webdav/webdav.go

```diff
@@ -43,7 +43,7 @@ import (
 )
 
 const (
-	minSleep      = 10 * time.Millisecond
+	minSleep      = fs.Duration(10 * time.Millisecond)
```

Comment:
This change ensures the setting is given the correct data type, as otherwise it wouldn't permit specifying the value using an `ms` suffix for instance.